Tests/Editor/Association/GameObjectsAssociationActivatorTest.cs

```diff
@@ -96,8 +96,10 @@ public void AwakeLogsWarningForMultipleActiveGameObjects()
 
             subject.Associations.Add(associationMock);
 
+            Debug.Log("This test is expecting a warning to be logged next.");
```

Comment:
What about instead of logging messages around the expect, what about using this to clear the log after the test has run?

https://docs.unity3d.com/ScriptReference/Debug.ClearDeveloperConsole.html

Reply:
I looked into this and I don't like it since it's an all-or-nothin approach. We want to hide that specific warning, not any others already in the console, nor any other logged during the run of the tests.

I tried setting our own log handler but it looks like there's no clean way to basically just filter what gets forwarded to Unity without introducing quite a bit of code just for that purpose.

Reply:
This may also be helpful
https://answers.unity.com/questions/707636/clear-console-window.html

Reply:
I'm fine with just logging this message then if that's the best of a bad situation.